utils/contentStyle.json

```
@@ -76,6 +76,14 @@
 						"correction": {
 							"replaceWith": "\\1\\2(\\3\\4)\\1"
 						}
+					},
+					{
+						"description": "accidental commonly doubled word",
+						"regex": "\\b(than|the|with|to|is|a|an|of|as|was|and|in)\\s+\\1\\b",
```

Double "is" can also be grammatically correct in some cases.

https://www.reddit.com/r/grammar/comments/y0vjfz/using_that_that_in_a_sentence_in_text/?rdt=52935
 
> "What it is is awful." - Here, both occurrences of "is" are 3rd person singular present-tense verb forms, and both are required.

---

Oops. I obviously should check my suggestions before typing them.

---

In this case it should probably be disambiguated with a comma, anyways?

`What it is is awful` -> `What it is, is awful`

---

... maybe not, but we might as well keep it in until somebody hits it as an actual problem

---

Well all it takes is one counter example to remove a word from this check. I'd say it doesn't matter if we've run into it in game, yet.
```suggestion
						"regex": "\\b(than|the|with|to|a|an|of|as|was|and|in)\\s+\\1\\b",
```